### PR TITLE
Update Design-wearable documents

### PR DESCRIPTION
**Pull request by s-k-song**
Resizing images with  tags in Design-wearable documents

Some images are separated and inserted into tables to be aligned with explanations

Video links are revised: previously, there was no icons displayed for the links.

### CODE REVIEW

File: docs/design/wearable/interaction/bezel-interactions.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ Bezel interactions refer to rotating the bezel to explore the screen. Users can
 
 |Interaction |    | Example |
 |------------|----|--------|
-|Turn the bezel clockwise|![](media/interaction_7.2.0_1.png)| Scroll right<br>Down<br>Increase<br>value<br>Accept<br>Zoom in|
-|Turn the bezel counterclockwise|![](media/interaction_7.2.0_2.png)|Scroll left<br>Up<br>Reduce value<br>Reject<br>Zoom out<br>|
+|Turn the bezel clockwise|<img src="media/interaction_7.2.0_1.png" width="128" height="91" />| Scroll right<br>Down<br>Increase<br>value<br>Accept<br>Zoom in|
```
Comment (journey2w) [inline docs/design/wearable/interaction/bezel-interactions.md]: Please remove `<br>` tag between "Increase" and "value".

**Comment by s-k-song** [comment]
As per the change request, the unnecessary \<br\> tag is deleted.

**Comment by s-k-song** [comment]
Video links are changed using \<video\> tags.
This commit is only for Design-wearable documents.

Video links in Design-mobile / TV documents will be changed in other branches.

**Comment by s-k-song** [comment]
Change more video links, which are missed in the last commit.

Some of them are inserted into a table, to be aligned for proper explanations over the do and don't bar images. (The media directory under the wearable/interaction directory is updated with the new images, too.)

**Pull request merged**